### DISCUSSION TITLE
Ozone Feature Request: Add Attachments to Reports

### DISCUSSION THREAD

**Discussion by rudyfraser** [bluesky-social/atproto#2977] [General Discussion]
It'd be great to be able to add attachments to reports. For example, a harassment vector I'm seeing now is new accounts mass DMing people and some of those folks are posting screenshots on the skyline. If a report comes in, a moderator might say "no evidence of harassment" because the account may not be publicly posting and people can't send images in reports and mods obviously can't see DMs. If someone sends or shares a screenshot of the harassment it would be good to come back and add that image to someones report in Ozone.

**Comment by Tamschi** [comment]
Small correction: You can report DMs via the three-dot menu there, by clicking "Report conversation".
Doing it that ways should reveal the conversation to the moderators.

In my experience, they (albeit scammers) got taken down fairly quickly.

**Reply by rudyfraser** [reply]
You cannot report a DM to 3rd party moderation services.

**Reply by Tamschi** [reply]
Ahh, I misunderstood. It's an interesting question, since e.g. in Germany it's usually not legal to forward private messages (whether directly or as screenshot).

Platforms usually require an exemption regarding moderation in their terms of service before they let someone use them.
I wonder if it would be possible to formulate one that covers third party moderation services also.

**Comment by bnewbold** [comment]
This would be a little funky: the blobs would get uploaded to the ozone instance? would need tight limits to prevent abuse.

But overall makes sense. Maybe base64-encoded so blobs can come along in a single HTTP POST?

**Reply by rudyfraser** [reply]
Not suggesting allowing people to send images when making reports (though that happened today with someone sending google drive links that were shown in Gmail's preview because I set up custom email notifications). 

Just that we as moderators can upload them directly to Ozone. This may be overloading Ozone into having a service desk functionality but IMO makes sense for how I do content moderation and how I have onboarded moderators.

There's already a "feature". I'd just want to "comment with attachment".
